mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
+python_version = 3.6
+warn_return_any = True
```

Comment:
Nice, I like this one. is there a good way to ignore certain folders (dev code, examples, etc) from the type checker tests?